src/server.js

```diff
@@ -10,7 +10,12 @@ app.engine('hbs', hbs({
   defaultLayout: 'main',
   layoutsDir: path.join(__dirname, './', 'views/layouts'),
   partialsDir: path.join(__dirname, './', 'views/partials'),
-  extname: 'hbs'
+  extname: 'hbs',
+  helpers: {
+    getMapLink: () => {
+      return `https://maps.googleapis.com/maps/api/js?key=${process.env.GOOGLE_API}&callback=_myMap`
```

Comment:
we can either make a file for this helper and test or we can i think if we call it somewhere it will be covered or just ignore the files in codecov but we dont want to do that